packages/theme/src/colors/index.ts

```diff
@@ -3,13 +3,22 @@
  *
  * {@link https://zeroheight.com/3ddd0f892/p/028ae9-colors/b/8982a2}
  */
-import alert from './alert'
-import primary from './primary'
-import secondary from './secondary'
+import alert, { AlertColor, alertColors } from './alert'
+import primary, { PrimaryColor, primaryColors } from './primary'
+import secondary, { SecondaryColor, secondaryColors } from './secondary'
 import token from './token'
 
-export { alert, primary, secondary, token }
+export const allColors: { [key: string]: string | (() => string) } = {
+  ...alertColors,
+  ...primaryColors,
+  ...secondaryColors,
+  ...token,
+}
 
+export type TokenColor = keyof typeof token
+export type Color = AlertColor | PrimaryColor | SecondaryColor | TokenColor
+
+export { alert, primary, secondary, token }
 export default {
```

Comment:
Can you remove the changes and spread the colors in `export default`?

```ts
export default {
  ...alert.critical,
  ...alert.informative,
  ...alert.success,
  ...alert.warning,
  ...primary.brand,
  ...primary.primaryBlue,
  ...secondary.secondaryBlue,
  ...secondary.secondaryDeepGreen,
  ...secondary.secondaryGold,
  ...secondary.secondaryLimeGreen,
  ...secondary.secondaryMintGreen,
  ...secondary.secondaryNeutralGrey,
  ...secondary.secondaryPeach,
  ...secondary.secondaryPeach,
  ...secondary.secondaryPink,
  ...token,
  alert,
  primary,
  secondary,
  token,
}
```

Reply:
Updated